### PR TITLE
Orm default number values

### PR DESCRIPTION
**Pull request by mmadariaga**
Numeric field default values should not be strings

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

File: library/Ivoz/Cgr/Domain/Model/TpDestinationRate/TpDestinationRateAbstract.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ abstract class TpDestinationRateAbstract
      * column: max_cost
      * @var string
```
Comment (Kaian) [inline library/Ivoz/Cgr/Domain/Model/TpDestinationRate/TpDestinationRateAbstract.php]: This field is defined as decimal in orm files. Should be documented as float.

File: library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Recording.RecordingAbstract.orm.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ Ivoz\Provider\Domain\Model\Recording\RecordingAbstract:
       precision: 10
       scale: 3
       options:
-        default: '0.000'
+        default: 0.000
```
Comment (Kaian) [inline library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Recording.RecordingAbstract.orm.yml]: Float fields have 0.000 as default, while decimal fields have 0.0
Is this intended?

Also precision and scale may not affect float fields.

Comment (mmadariaga) [reply library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/Recording.RecordingAbstract.orm.yml]: It turns out that you can specify float precision `duration float(10,3) NOT NULL DEFAULT '0.000'`. I'll set 0.0 as default value

**Pull request merged**